### PR TITLE
Create inversion specification files for NAME-RHIME

### PR DESCRIPTION
**Pull request by alexdanjou**

### CODE REVIEW

File: inversion_specifications/NAME-RHIME/overview.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+### Model and measurement uncertainties
+
+The measurement are averaged on a four hour basis. We also remove the data for which the PBL height is less than 50m different from the inlet height.
+
+The inversion is done by comparing the observation vector ($Y_{obs}$) to the simulated observations given by:
+
+$$
+Y_{mod} = H_x x + H_{bc}x_{bc} + \epsilon
+$$
+
```
Comment (mrghg) [inline inversion_specifications/NAME-RHIME/overview.md:41]: @alexdanjou, please can you use Latex math notation in the rest of this section, as I've done here. This should make it easier to follow

File: inversion_specifications/NAME-RHIME/overview.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+NAME model (version 7.2) developped by the UK Met Office (Muller et al. 2013), a Langrangian particle model that we run backwards in time for each measurement site used in the inversion.
+
+### Scope
+
+- Domain: 10.73°N to 79.06°N and from 97.9°W to 39.38°E.
+- Period: 2018-01-01 to 2023-01-01
+
+### Meteorology
+
+The meteorological fields from the UK Met Office Unified Model are used. The model is run on a grid of horizontal resolution 0.0135° x 0.0135° over the UK (Manning and Redington et al. 2021). Vertically, it uses 57 levels, expanding to 12.2km.
```
Comment (mrghg) [inline inversion_specifications/NAME-RHIME/overview.md]: The values you've quoted are not consistent with those in Table 1 of that paper. You need to note that the horizontal resolution of the "global" met has changed with time.

Comment (mrghg) [reply inversion_specifications/NAME-RHIME/overview.md]: Also, should be referenced as Manning et al. (2021), no?

File: inversion_specifications/NAME-RHIME/ch4.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# RHIME specification for CH4
+
+Use this file to provide any species-specific inversion specifications, or modifications to the general approach outined in the overview.
```
Comment (mrghg) [inline inversion_specifications/NAME-RHIME/ch4.md:3]: Remove these comments from the template

File: inversion_specifications/NAME-RHIME/ch4.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+## Fluxes
+
+List of flux datasets, including any modifications:
+
+- EDGAR combined with UKGHG for UK.
+  - Reference:
+    - EDGAR version 8 : European Union 2023, European Commission, Joint Research Centre (JRC), EDGAR (Emissions Database for Global Atmospheric Research) Community GHG database, comprising IEA-EDGAR CO2, EDGAR CH4, EDGAR N2O and EDGAR F-gases version 8.0 (2023)
+    - UKGHG inventory for the year 2019.
+  - Native spatial resolution: 0.1degree x 0.1degree for EDGAR, 1km x 1km for UKGHG
```
Comment (mrghg) [inline inversion_specifications/NAME-RHIME/ch4.md]: Use the degree symbol

File: inversion_specifications/NAME-RHIME/overview.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+### Model input parameters
+
+The langragian transport model is used with a release of 20000 particles/hour. Particles are released from the measurement altitude ± X m. Particles are tracked for 30 days prior to the measurement, or until they leave the domain.
```
Comment (mrghg) [inline inversion_specifications/NAME-RHIME/overview.md]: can you clarify how the particles are distributed vertically (and horizontally?)? I think they are released from a vertical line centred on the inlet, plus or minus some number of metres. But I may be wrong, and they could be released from a point. Can you check?

Comment (mrghg) [reply inversion_specifications/NAME-RHIME/overview.md]: Here, we also need to clarify what has been used to define the "surface" for the purposes of calculating the footprint. I **think** (but again, please check) that we assume that particles are considered to have interacted with the surface if they are within 40m of the ground

**Review by mrghg — CHANGES_REQUESTED** [review]

File: inversion_specifications/NAME-RHIME/overview.md (hunk 2 above)
Comment (alexdanjou) [reply inversion_specifications/NAME-RHIME/overview.md]: I took the last line of table 1, which give the resolution for UK and Ireland. I realize that as we use whole Europe it's indeed global that matter, right? And we do our runs for 2018 to 2023, so the second-to-last should be quoted, right ? (July 2017 to present, horizontal : 0,141 x 0.0938 degrees, 59 vertical levels up to 29km)

For the ref, it said that the two first authors contributed equally so didn't really know how to handle that. But if you confirm I'll put Manning 2021.

Comment (mrghg) [reply inversion_specifications/NAME-RHIME/overview.md]: > I took the last line of table 1, which give the resolution for UK and Ireland. I realize that as we use whole Europe it's indeed global that matter, right? And we do our runs for 2018 to 2023, so the second-to-last should be quoted, right ? (July 2017 to present, horizontal : 0,141 x 0.0938 degrees, 59 vertical levels up to 29km)

Yes, that's right
 
> For the ref, it said that the two first authors contributed equally so didn't really know how to handle that. But if you confirm I'll put Manning 2021.

Ah, good point

Comment (mrghg) [reply inversion_specifications/NAME-RHIME/overview.md]: Sorry, I just realised that I didn't understand your question correctly. We embed the UKV meteorology within the global met fields, so we need to reference both

**Review by mrghg — APPROVED** [review]
I've made some final edits and am going to merge this in

**Pull request merged**